terraform/variables.tf

```
@@ -0,0 +1,34 @@
+variable "app_name" {
+  description = "Application name"
+  type        = string
+  default     = "mlflow-server"
+}
+
+variable "channel" {
+  description = "Charm channel"
+  type        = string
+  default     = null
```

```suggestion
  default     = "2.15/stable"
```
based on our discussion with @orfeas-k